pytket/docs/getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting Started
+
+The tket compiler is a powerful tool for optimizing and manipulating
+platform-agnostic quantum circuits, focused on enabling superior performance on
+NISQ (noisy intermediate-scale quantum) devices. The pytket package provides an
+API for interacting with tket and transpiling to and from other popular quantum
+circuit specifications.
+
+Pytket is compatible with 64-bit Python 3.10, 3.11, 3.12 and 3.13, on Linux,
+MacOS (13.0 or later) and Windows. Install pytket from PyPI using:
+
+```
+pip install pytket
+```
+
+This will install the tket compiler binaries as well as the pytket package. For
+those using an older version of pytket, keep up to date by installing with the
+`--upgrade` flag for additional features and bug fixes.
+
+There are separate packages for managing the interoperability between pytket and
+other quantum software packages which can also be installed via PyPI. For
+details of these, see the
+[pytket extensions](extensions.html) documentation.
+
+The quantum circuit is an abstraction of computation using quantum resources,
+designed by initializing a system into a fixed state, then mutating it via
+sequences of instructions (gates).
+
+The native circuit interface built into pytket allows us to build circuits and
+use them directly.
+
+```
```

Comment:
Minor thing but for code snippets like this you can do the following



"```{code-cell} ipython3

from pytket import Circuit

trivial_circ = Circuit()        # no qubits or bits
quantum_circ = Circuit(4)       # 4 qubits and no bits
mixed_circ   = Circuit(4, 2)    # 4 qubits and 2 bits
named_circ   = Circuit(2, 2, "my_circ")


This will match the styling of the manual examples once the msyt fix is in effect.


For pages with cell executions you need to add this metadata to the top of each page

```
---
file_format: mystnb
kernelspec:
  name: python3
---
```

Reply:
I have added these in but made the cells skip execution because they would use a qasm files that doesn't exist or qiskit and pytket-qiskit that aren't installed (not sure how the build system works with environments - would I have to install them with poetry to get them to build?)

Reply:
Yes you would need to install pytket-qiskit with poetry for this to be runable. I think `skip-execution` is the way to go here as I'd prefer to not introduce a pytket-qiskit dependency.